builder/builder.go

```diff
@@ -161,6 +162,23 @@ func (b *Builder) createRpmDirs() error {
 	return nil
 }
 
+// Get latest CLR version
+func (b *Builder) getLatestUpstreamVersion() (string, error) {
+	resp, err := http.Get(b.UpstreamURL + "/latest")
```

Comment:
This is basically just a path. I would prefer to use `path.Join` here instead of `"somethingThatMightEndIn/" + "/..."`

Reply:
I had the same thought, but read people saying that path.Join can mangle URLs, and they wanted me to import the `net/url` package and use a far more complicated `ResolveReference` method.  I figured I'd avoid the import and complication for now.

Think I should change it?

Reply:
I'll give it a go. It's "the right thing to do".

Reply:
My experience was the same as reported by Kevin two comments above.

Reply:
That was easy enough.